Sources/ElasticSwift/Requests/DeleteRequest.swift

```diff
@@ -41,113 +45,57 @@ public class DeleteRequestBuilder: RequestBuilder {
         return self
     }
     
-    public func set(completionHandler: @escaping (DeleteResponse?, Error?) -> Void) -> Self {
+    public func set(completionHandler: @escaping (DeleteResponse?, Error?) -> ()) -> Self {
```

Comment:
Though, `Void` is a type alias for `()` we should use `Void` explicitly where the function should/does not return.